climakitae/core/data_interface.py

```diff
@@ -991,7 +995,24 @@ def _update_scenarios(self):
             if scen in historical_scenarios
         ]
         self.param["scenario_historical"].objects = scenario_historical_options
-        if self.scenario_historical not in scenario_historical_options:
+
+        # test to check if one list is contained within another as python doesn't have that out of the box
+        def _check_list_contained(a, b):
+
+            # convert list a to numpy array
+            a_arr = np.array(a)
+            # convert list b to numpy array
+            b_arr = np.array(b)
+
+            for i in range(len(b_arr)):
+                if np.array_equal(a_arr, b_arr[i : i + len(a_arr)]):
+                    return True
+            return False
+
+        # check if new selection has the historical scenario options and if not select the first new option
+        if _check_list_contained(_scenario_historical, scenario_historical_options):
```

Comment:
Is it possible to use sets here for comparing these two lists since both of these can only be a max length of 2 (and since we know what all the possible values)? 

Like `if set(_scenario_historical).issubset(scenario_historical_options):` ? Or is this logic too simplified?

Reply:
Yeah let me figure out how to do this without `numpy` silly to import it just for this

Reply:
I think I have fixed the function in [5401d61](https://github.com/cal-adapt/climakitae/pull/393/commits/5401d61950de5db57994917c9415197a7ed978f4)